workloads/files/workload-concurrent-scale-up-down-script-cm.yml

```diff
@@ -0,0 +1,107 @@
+      echo "" > /var/lib/pbench-agent/tools-default/perf
+      echo "" > /var/lib/pbench-agent/tools-default/pidstat
+      echo "" > /var/lib/pbench-agent/tools-default/sar
+      master_nodes=`oc get nodes -l pbench_agent=true,node-role.kubernetes.io/master= --no-headers | awk '{print $1}'`
+      for node in $master_nodes; do
+        echo "master" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+      infra_nodes=`oc get nodes -l pbench_agent=true,node-role.kubernetes.io/infra= --no-headers | awk '{print $1}'`
+      for node in $infra_nodes; do
+        echo "infra" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+      worker_nodes=`oc get nodes -l pbench_agent=true,node-role.kubernetes.io/worker= --no-headers | awk '{print $1}'`
+      for node in $worker_nodes; do
+        echo "worker" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+    fi
+    source /opt/pbench-agent/profile
+    workload_log "Done configuring pbench Concurrent Scale Up Down"
+
+    workload_log "Running Concurrent Scale Up Down workload"
+    if [ "${PBENCH_INSTRUMENTATION}" = "true" ]; then
+      pbench-user-benchmark -- sh /root/workload/workload.sh
+      result_dir="/var/lib/pbench-agent/$(ls -t /var/lib/pbench-agent/ | grep "pbench-user" | head -1)"/1/sample1
+      if [ "${ENABLE_PBENCH_COPY}" = "true" ]; then
+        pbench-copy-results --prefix ${CONCURRENT_SCALEUP_TEST_PREFIX}
+      fi
+    else
+      sh /root/workload/workload.sh
+      result_dir=/tmp
+    fi
+
+    workload_log "Completed Concurrent Scale Up Down workload run"
+
+    workload_log "Checking Test Results"
+    workload_log "Checking ansible-playbook scale_up_complete.yaml execution exit code : ${exit_code}"
+
+    if [ "$(jq '.exit_code==0' ${result_dir}/exit.json)" = "false" ]; then
+      workload_log "Concurrent Scale Up Down Failure"
+      workload_log "Test Analysis: Failed"
+      exit 1
+    fi
+    # TODO: Check pbench-agent collected metrics for Pass/Fail
+    # TODO: Check prometheus collected metrics for Pass/Fail
+    workload_log "Test Analysis: Passed"
+
+  workload.sh: |
+    #!/bin/sh
+    set -o pipefail
+
+    result_dir=/tmp
+    if [ "${PBENCH_INSTRUMENTATION}" = "true" ]; then
+      result_dir=${benchmark_results_dir}
+    fi
+
+    # Save a local copy of the read-only ~/.kube/config
+    cp ${KUBECONFIG} /tmp/kube_config
+    ls -ltr /tmp
+    export KUBECONFIG=/tmp/kube_config
+    echo ${KUBECONFIG}
+
+    # git clone svt repo in /root
+    cd /root
+    git clone https://github.com/openshift/svt.git
+    cd svt
+    git status
+    cd /root/svt/openshift_performance/ci/content
+    ls -ltr
+   
+    start_time=$(date +%s)
+    my_time=$(date +%Y-%m-%d-%H%M)  
+
+    ansible-playbook -vv -e api_url=${CONCURRENT_SCALEUP_API_URL} -e login_username="kubeadmin" -e login_passwd=${CONCURRENT_SCALEUP_KUBEADMIN_PASSWD} scale_up_complete.yaml 2>&1 | tee /tmp/output_scale_up_complete-${my_time}.log
```

Comment:
Think we need to parameterize the login username as well?

Reply:
@wabouhamad Suggest updating the PR to use my new changes which take advantage of existing KUBECONFIG and which avoid the need for login